recipes/compyle/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "compyle" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: "./pypr-compyle-v0.5-112-g42a7868.tar.gz"
+  sha256: "eceb3bc0970b570056138174e77a2eb0d6be2128c81a7315436bb996c668bbd3"
```

Writing the hash predictably makes it easier for the update bot:
```suggestion
  sha256: eceb3bc0970b570056138174e77a2eb0d6be2128c81a7315436bb996c668bbd3
```